project-koku/koku
# [COST-4168] null storage usage

## Jira Ticket

[COST-4168](https://issues.redhat.com/browse/COST-4168)

## Description

This change adds coalesce to prevent returning `null` for usages.

## Testing

1. on main, but using the nise static yaml in this PR, `make create-test-customer; make load-test-customer-data test_source=AWS` using this version of Nise: https://github.com/project-koku/nise/pull/465
2. go to: http://localhost:8000/api/cost-management/v1/reports/openshift/volumes/?group_by[project]=storage-only
3. see: 
```
"usage": null,
```
5. Checkout this branch
6. Restart Koku
7. go back to: http://localhost:8000/api/cost-management/v1/reports/openshift/volumes/?group_by[project]=storage-only
8. see:
```
        "total": {
            "usage": {
                "value": 0.0,
                "units": "GB-Mo"
            },
```

## Notes

...
